packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/MarkerId.java

```diff
@@ -0,0 +1,43 @@
+package io.flutter.plugins.googlemaps;
+
+/**
+ * Represents the user assigned MarkerId.
+ *
+ * <p>To be kept in sync with MarkerId on dart side.
+ */
+public class MarkerId {
```

Comment:
I'd just keep the marker IDs as Strings in the Android implementation, saves us some boilerplate.

Reply:
I was thinking the same, I was hoping that having a type here will let us distinguish between dart marker id and google map api's marker id (which we are keeping as a string).

Reply:
I think the name should cover that well enough, it's used in a relatively narrow scope anyway...